Assets/MRTK/Providers/Oculus/XRSDK/Editor/OculusXRSDKConfigurationChecker.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.MixedReality.Toolkit.XRSDK.Oculus
     [InitializeOnLoad]
     static class OculusXRSDKConfigurationChecker
     {
-        private const string AsmDefFileName = "Microsoft.MixedReality.Toolkit.Providers.XRSDK.Oculus.asmdef";
+        private const string AsmDefFileName = "Oculus.asmdef";
```

Comment:
This naming seems a bit risky since I can imagine it colliding with another .asmdef at some point, though you've said you've said you'll be prepending these with MRTK so it should be good!

Reply:
fixed by appending MRTK